### DISCUSSION TITLE
ule that a channel can only be assigned to one room

### DISCUSSION THREAD

**Discussion by ACX-Com** [home-assistant/architecture#1019] [Entity Models]
# Deutsch:
Es gäbe mehrere Gründe dies zuzulassen.
Beispiel 1:
Ein Aktor steuert die Umwälzpumpe für Warmwasser.
Dieser Aktor sollte jedem Raum mit Warmwasser-Entnahme zugeordnet werden können.
Beispiel 2:
Ein Doppel-Relais steuert die Fußbodenventile für zwei Räume (je ein Kanal je Raum).
Auch hier ist die Zuweisung EINES Geräts zu den beiden Räumen sinnvoll.

Des Weiteren kann man unter Räume auch Etagen erstellen (diese gibt es sonst nicht).
Hier ist es oft praktisch, z.B. Rauchmelder:
Eine Etage hat mehrere Räume und diese je einen oder mehrere Rauchmelder.
Um alle Rauchmelder sowohl jeweils in ihrem Räumen, aber gesammelt auch in einer Etage zu verwalten/anzuzeigen, wäre es sinnvoll, die Kanäle der Rauchmelder mehreren "Räumen" zuordnen zu können.

# English:
There are several reasons to allow this.
Example 1:
An actuator controls the circulation pump for hot water.
This actuator should be able to be assigned to every room with hot water extraction.
Example 2:
A double relay controls the floor valves for two rooms (one channel per room).
Here too, assigning ONE device to both rooms makes sense.

You can also create floors under rooms (these don't exist otherwise).
This is where it often comes in handy, e.g. smoke detectors:
A floor has several rooms, each with one or more smoke detectors.
In order to manage/display all smoke detectors both in their rooms and collectively on one floor, it would make sense to be able to assign the smoke detector channels to several "rooms".

**Comment by frenck** [comment]
Hi there @ACX-Com 👋 

I think this is a discussion or feature request and not a technical proposal/implementation for which you are looking for approval.

Please use our community forums to discuss features of Home Assistant instead.

Thanks! 👍 

../Frenck
